Assets/Treasure/TDK/Runtime/Common/Constants.cs

```diff
@@ -62,7 +62,11 @@ public static class Constants
                 }
             },
             {
-                ChainId.TreasureTopaz, new Dictionary<Contract, string> {}
+                ChainId.TreasureTopaz, new Dictionary<Contract, string> {
+                    { Contract.Magic, "0x095ded714d42cbd5fb2e84a0ffbfb140e38dc9e1" },
```

Comment:
This is WMAGIC, better to not include it in the SDK, just the Magicswap example. MAGIC is the native token and has no token address